src/Components/DynamicFields.res

```diff
@@ -19,7 +19,10 @@ let make = (
     PaymentMethodsRecord.getPaymentMethodTypeFromList(
       ~list,
       ~paymentMethod,
-      ~paymentMethodType,
+      ~paymentMethodType=PaymentUtils.getPaymentMethodName(
```

Comment:
This method as I saw was removing the "transfer" part correct? Doing a lookup after removing doesn't seem optimal right? Also this isnt used for display is it?